src/content/docs/basics/intercepting-requests.mdx

```diff
@@ -195,7 +195,20 @@ export const handlers = [
 
 ### How do I use relative URLs in Node.js?
 
-You don't. In Node.js, there is nothing to be relative to. If you are describing a network behavior for a Node.js application, use the absolute URLs you are requesting. If you are using MSW with a Node.js-based test runner, like Jest or Vitest, configure those runners accordingly to support relative URLs (i.e. polyfill the `window.location` object).
+You don't. In Node.js, there is nothing to be relative to. If you are describing a network behavior for a Node.js application, use the absolute URLs you are requesting. If you are using MSW with a Node.js-based test runner, like Jest or Vitest, configure those runners accordingly to support relative URLs (i.e. polyfill the `window.location` object), 
+
+### How to set `baseURL` in Node.js?
+Create a custom high-order function if you wish to reuse the same base path for multiple handlers. Like so:
+
+``` ts
+const github = (path: string) => {
+  return new URL(path, 'https://github.com').toString()
```

Comment:
I'd recommend using `.href` instead of `.toString()`. It's shorter and result into the same URL string being returned.